content/chronograf/v1.9/about_the_project/release-notes-changelog.md

```diff
@@ -8,6 +8,32 @@ menu:
     parent: About the project
 ---
 
+## v1.9.3 [2022-02-02]
+
+{{% note %}} **NOTE:** We did not release version 1.9.2 due to a bug that impacted communication between the browser’s main thread and background workers.  This bug has been fixed in the 1.9.3 release.
```

Comment:
@lesam, to be sure, can you confirm whether the `Key not found in database` issue also affected in 1.9.1? ....saw [discussion here](https://github.com/influxdata/EAR/issues/2944#issuecomment-1021311804) 
thank you!

Reply:
it did not.  It was 1.9.2 only.